java/ql/test/library-tests/frameworks/guava/TestBase.java

```diff
@@ -60,4 +60,4 @@ void test3() {
     void test4() {
         sink(Preconditions.checkNotNull(taint())); // $numTaintFlow=1
     }
-}
```

Comment:
Stray whitespace change

Reply:
Sure, but the end-of-file newline was missing. My editor tends to fix that as it's otherwise not a proper text-file (also highlighted by the "no newline at end of file"-icon that github shows).